kube-runtime/src/events.rs

```diff
@@ -1,13 +1,23 @@
 //! Publishes events for objects for kubernetes >= 1.19
+use std::hash::{Hash, Hasher};
+use std::{collections::HashMap, sync::Arc};
+
 use k8s_openapi::{
-    api::{core::v1::ObjectReference, events::v1::Event as K8sEvent},
+    api::{
+        core::v1::ObjectReference,
+        events::v1::{Event as K8sEvent, EventSeries},
+    },
     apimachinery::pkg::apis::meta::v1::{MicroTime, ObjectMeta},
-    chrono::Utc,
+    chrono::{Duration, Utc},
 };
+use kube_client::ResourceExt;
 use kube_client::{
-    api::{Api, PostParams},
+    api::{Api, Patch, PatchParams, PostParams},
     Client,
 };
+use tokio::sync::RwLock;
+
+const EVENT_FINISH_TIME: Duration = Duration::minutes(6);
```

Comment:
_TIME is perhaps misleading for a Duration unit.

```suggestion
const CACHE_TTL: Duration = Duration::minutes(6);
```

or perhaps something else.

Reply:
Fixed in bebbb103967e201d7a393ae1865f4e21bcd72c42